tests/system/action/motion/test_create.py

```diff
@@ -107,9 +111,37 @@ def test_create_simple_fields(self) -> None:
             },
         )
         self.assert_status_code(response, 200)
-        self.assert_model_exists(
+        motion = self.assert_model_exists(
             "motion/2", {**motion, "attachment_meeting_mediafile_ids": [80]}
         )
+        assert motion.get("submitter_ids") is None
+
+    def test_create_additional_submitter_forbidden(self) -> None:
+        self.set_models(
+            {
+                "motion/1": {
+                    "title": "title_eJveLQIh",
+                    "meeting_id": 1,
+                },
+                "meeting/1": {"mediafile_ids": [8], "meeting_mediafile_ids": [80]},
```

Comment:
Media files and motion/1 shouldn't be necessary.

Reply:
Done